book/creating-a-basic-website/creating-a-twig-template.rst

```diff
@@ -0,0 +1,144 @@
+Creating a Twig Template
+========================
+
+Twig is an awesome option for rendering HTML. It got some nice features like
+blocks and inheritance. That's why we use and love Twig.
+
+.. note::
+	It is also possible to use plain PHP or any other template lannguage like Smarty,
+	but we strongly recommend Twig.
```

Comment:
Theoretically that should be possible, but I have never tested it. So I am not sure if it is a great idea to mention that here.